react-ui/src/components/LogInForm.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component} from "react";
+import { Container, Row, Col } from 'react-grid-system';
+import { Redirect, Route, withRouter } from "react-router-dom";
+import Title from "./Title"
+import "./LogInForm.css"
+import dashboard from "./Layouts/Dashboard"
+
+class logInForm extends Component {
```

Comment:
This PR seems to contain the work for login UI story? Is this intended?

Reply:
It is needed to build subsequent pages, so I will update my PR comment to include the log-in UI set up as well.

Reply:
If this PR also encompasses the login UI setup, remember to validate the user credentials using the appropriate backend endpoint `api/login` and display an error if null is returned from the endpoint, i.e., the user credentials are invalid.

Reply:
This ticket encompasses  S3T2 and General T2, but neither of them are fully fleshed out. This just needs to be passed so that other people can work on the front-end in the meanwhile.